apps/iotune/iotune.cc

```diff
@@ -143,11 +145,15 @@ struct evaluation_directory {
             iotune_logger.error("Error while parsing sysfs. Will continue with guessed values: {}", se.what());
             _max_iodepth = 128;
         }
+        if (_force_io_depth) {
+            _max_iodepth = *_force_io_depth;
```

Comment:
I guess with _force_io_depth == 0 the whole thing will get stuck ... (see my other comment below for continuation of this guess)